migrations/versions/0393_add_go_live_template.py

```diff
@@ -0,0 +1,101 @@
+"""
+
+Revision ID: 0393_add_service_go_live_template
+Revises: 0392_go_live_cols_non_nullable
+Create Date: 2022-12-28 12:12:12.929661
+
+"""
+import textwrap
+
+from alembic import op
+from flask import current_app
+
+revision = "0393_add_go_live_template"
+down_revision = "0392_go_live_cols_non_nullable"
+
+
+template_id = current_app.config["ORGANISATION_HAS_NEW_GO_LIVE_REQUEST_TEMPLATE_ID"]
```

Comment:
In other migrations we've hardcoded values like these and the Notify service and user ID instead of importing them so that the migrations still work if we re-arrange the code. Could be worth doing here too (but I'll leave that decision up to you)

Reply:
```suggestion
template_id = "5c7cfc0f-c3f4-4bd6-9a84-5a144aad5425"
```